config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -508,8 +508,8 @@ periodics:
         value: '{"AllAlpha":true,"EventedPLEG": false}'
       - name: RUNTIME_CONFIG
         value: '{"api/alpha":"true", "api/ga":"true"}'
-      - name: FOCUS
-        value: "."
+      - name: LABEL_FILTER
+        value: !Slow && !Disruptive && !Flaky && Feature: isSubsetOf Alpha
```

Comment:
https://github.com/kubernetes-sigs/kind/pull/3582 got merged. Does that mean that it is usable now?

In other words, can we merge this PR?

/cc @aojea @BenTheElder